src/main/java/com/schematica/world/storage/Schematic.java

```diff
@@ -73,6 +75,7 @@ private boolean hasOffset()
 
     //MINECOLONIES END
 
+    @SuppressWarnings ("deprecation")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)